tox.ini

```diff
@@ -8,7 +8,6 @@ requires=tox-conda
 [base]
 deps =
     :preinstall: -rrequirements.txt
-    -rrequirements-igwn.txt
```

Comment:
Because I think some of our tests need it.

Reply:
@spxiwh If that's the case, I think we may have made a mistake in the requirements files and I can move them over. The point of the IGWN file is to just have the requirements needed to access LDG specific services (and hence wouldn't be runnable from the CI anyway).